docs/use_database.md

```diff
@@ -0,0 +1,3 @@
+# Creating/managing databases
```

Comment:
Leave it out and create a story to develop the missing parts.

Reply:
added basic info here too.